wiki/en/en-Installation-for-Windows.md

```diff
@@ -28,14 +28,14 @@ Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) to
 
 For ASIO4ALL ([download](http://www.asio4all.org){: target="_blank" rel="noopener noreferrer"}), you may need to experiment a bit to find the right configuration since every computer is a bit different.
 
-**Tip:** Setup your sound card while you're [connected to a server](Onboarding#2-connecting-to-a-server) to hear your instrument or voice and check if everything is correctly setup, but first read on.
+**Tip:** Set up your sound card while you're [connected to a server](Onboarding#2-connecting-to-a-server) to hear your instrument or voice and check if everything is correctly set up; but first read on.
 
```

Comment:
Tip: install the ASIO4All "Off-line Settings" panel.

Reply:
I think this needs more of an explanation. I don't use Windows so don't really know what this means/does.

Reply:
It's part of the ASIO4ALL installer options.

Reply:
If it's part of the installer options then I think we should either mention it further up, in the "Download and install an ASIO driver" section, or get rid of that first link to it (it's repeated again -and again- a few lines below; I don't see the point in linking to it all over the place on the same page), and point this option out when the link is provided. Otherwise people might just click on the first link they see and install it, only to later have to uninstall/reinstall it.